doc/nrf/gs_toolchain.rst

```diff
@@ -0,0 +1,19 @@
+.. _gs_toolchain:
+
+Installing the |NCS| from a zip (experimental)
```

Comment:
See comment to line 10. My proposal is to include Windows in the title. This will make it long, but "experimental" is bound to disappear sooner or later.